Customize flip and background touch behavior

simolus3 opened the pull request:
As discussed in #7, this PR introduces an option to turn background touch-blocking off. It also allows users to implement a custom flip behavior that doesn't flip whenever the card is touched. I've added some tests and updated the readme accordingly.

Fixes #8

simolus3 committed:
Customize flip and background touch behavior

simolus3 committed:
typo

simolus3 committed:
also add example in docs

feibyte commented on lib/flip_card.dart:
```diff
@@ -42,17 +41,54 @@ class AnimationCard extends StatelessWidget {
       this.onFlip,
-      this.direction = FlipDirection.HORIZONTAL})
+      this.direction = FlipDirection.HORIZONTAL,
+      this.blockInactiveInteractions = true,
```
I reckon you are right.  We don't need this flag.  Can't think of any cases.

simolus3 replied:
Alright, I've removed it.

feibyte commented on lib/flip_card.dart:
```diff
@@ -115,30 +155,43 @@ class FlipCardState extends State<FlipCard> with SingleTickerProviderStateMixin
+    // if we need to flip the card on taps, wrap the content
+    if (widget.flipOnTouch) {
+      return GestureDetector(
+        behavior: HitTestBehavior.translucent,
```
Nice catch!

feibyte commented on README.md:
````diff
@@ -28,3 +28,26 @@ FlipCard(
 );
 ```
 
+You can also configure the card to only flip when desired by using a `GlobalKey` to
````
Great work!
Could you pls help to maintain the changelog file?  Or I could amend it later.

simolus3 replied:
I've added a broad description of the changes to the changelog and bumped the version number in the pubspec file.

simolus3 committed:
Address review comments

- removed blockInactiveInteractions parameter, now always on
- described changes in changelog
